modules/recon/companies-domains/viewdns_reverse_whois.py

```diff
@@ -0,0 +1,36 @@
+from recon.core.module import BaseModule
+from lxml.html import fromstring
+from urllib.parse import quote_plus
+
+class Module(BaseModule):
+
+	meta = {
+		'name': 'viewdns.info reverse whois domain harvester',
+		'author': 'Gaetan Ferry (@_mabote_) from @synacktiv',
+		'version': '1.0',
+		'description': 'Harvests domain names belonging to a company by using '
+					+  'the viewdns.info free reverse whois tool.',
```

Comment:
Random question... where is this in the spec/PEP? I have PRs with single-line string wrapping done several different ways now and I cant find documentation on how its supposed to be done in Python 3.

Reply:
@lanmaster53  These are the only ones that I could find https://www.python.org/dev/peps/pep-0295/ https://www.python.org/dev/peps/pep-3126/ https://www.python.org/dev/peps/pep-3101/

Reply:
Heh, 2 of those were rejected PEPs. I think this is about as good as we're going to get. https://www.python.org/dev/peps/pep-0008/#maximum-line-length

So here, @mab0te is basically doing the accepted math wrap with the line break before the operator (`+`) and concatenating multiple strings together to stay under 79 chars. Cool. Looks like PEP 8 approves. Otherwise, PEP 8 prefers wrapping in a container which I don't like for strings. I think it's confusing to read. PEP 8 speaks against using backslashes actually, which surprised the heck out of me as that's the most common one I see. I wonder if there is a performance hit using the adding approach vs. the contained approach. Hm...